extension-admob/ext.manifest

```diff
@@ -3,7 +3,7 @@ name: "AdMobExt"
 platforms:
     android:
         context:
-            aaptExtraPackages: ["androidx.work", "com.google.android.gms.common", "androidx.startup"]
+            aaptExtraPackages: ["androidx.work", "com.google.android.gms.common", "androidx.startup", "androidx.lifecycle"]
```

Comment:
Do we still need these? I though we generate them now.

Reply:
Hmm, let me try without them.

Reply:
Yep, no need anymore!